CommandHelper/LocalPackages/commands/buffs/rasalghul.ms

```diff
@@ -0,0 +1,24 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': "§2Ra's §eal §2Ghul",
+            'class': 'rasalghul',
+            'sig': 'o19MoXCpaJRMAPlw+/gRgeNm7sJBA90EtCuIoWODN9JUzPPc2rX073oZRVZktsUzYsdLBGiu9ARB5EgYtoHg5pTUCgVBmmkYWX4Tm3zELLpWD7h/f6WD/CfPUV2f+Yug0BHeQRZIuwvKtHltL5sEfabCz6gOB4DLHbBpgTOOjsVWTlhByAykqySl9D0CgA91y5JPe0QgIlgLmDcd8j8EYdbYNDH7b6CHysPqf0hodTWc7W49M15AFSujje9RbuwJ/RTEpgOzIF1UtodfINxra3kFbpFEbrq4V6Zykb5Ca+WDxphq40NYOe6CnvlzdsrfvpFr/ZIE20tBApHTXKp7MtxvlQFB9hdskNzyh20dzGEu4njIgjF6BTIc67kyIVonxTS47rJ4sk0iEyDEm3RI9tpngwyvIXllXLR2JbWhQnf43xQuBxY14UmFsuCA3lHU4rnlgyWJXjSzt/WpQBHmo5eWyd2+k0OHMFjIOWlCG2hPo2Tc6sMcKOFYiAU+U/itGQRO6fkrzaMsK314Q/5czcbQMiyb9DGlSroqmw8ASXJb7/638zBgIlCaKTOXpIrY/sB79D7YHkp0G85tsv+n9vg+jhMmu3LivZcsMNnAnCxmtH/A8kLeEeBB3gLrAr9RnEwpAkkrRbY1+qCjDkpndswyWukMelUxSeBoBPAJ84U=',
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODU4ODQ4MjE2NjcsInByb2ZpbGVJZCI6IjI4YmM1NTllZjZiYzQxZDVhOWE4NGIxNzZiMGM2ZDBhIiwicHJvZmlsZU5hbWUiOiJNaXN0b2ZmZWxlczAxIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS80Yjk4MzhlNjdhYWU3MWZlZTkxYjY5NzYzOWIwZTE0YjY4NGY5OTRjYjExZWUzY2VkMTE1N2QwODMyZTI1In19fQ==',
+            'trail': '',
+            'op': 'false',
+            'speech': "@plr.", I am the master of fighting techniques",
```

Comment:
I think you deleted somthing here cuz now the " are not balanced. It was fine before :(